### PR TITLE
Bugfix/pro 3484/avoid build unzip

### PR DESCRIPTION
**Pull request by patrickvacek**

### CODE REVIEW

**Comment by patrickvacek** [comment]
Still dealing with a provisioning bug here, but getting much closer.

File: tests/config_tests_prov_bad.toml [hunk 1]
```diff
@@ -1,35 +0,0 @@
-[gateway]
```
Comment (patriotyk) [inline tests/config_tests_prov_bad.toml]: Why You removed this file?

Comment (patrickvacek) [inline tests/config_tests_prov_bad.toml]: It doesn't appear to be used anywhere. Am I wrong? If we are using it, I of course have no problem keeping it!

**Comment by patrickvacek** [comment]
The provisioning bug is just an issue of not having 'certificates_directory' in the [tls] section, which is not part of aktualizr. So this pull request is finally ready for review.

**Comment by cajun-rat** [comment]
Looks great.  Note that this requires matching changes to meta-updater

**Pull request merged**